src/settings/defaults.js

```diff
@@ -353,6 +353,55 @@ function ticketTypeSettingDefault (callback) {
   })
 }
 
+/**
+ * Sets default status of tickets during creation.
+ * @param {Parameters<async.AsyncFunction>[0]} callback 
+ */
+function ticketStatusSettingDefault(callback) {
+  const statusSettingName = 'ticket:status:default'
+  callback = _.isFunction(callback) ? callback : () => {}
+  SettingsSchema.getSettingByName(statusSettingName, function(err, setting) { 
+    if (err) {
+      winston.warn(err)
+      return callback(err)
+    }
+
+    if (setting) {
+      return callback()
+    }
+
+    const ticketStatusSchema = require('../models/ticketStatus')
+    ticketStatusSchema.getStatus(function(err, statuses) {
```

Comment:
Can you make large call back functions to arrow functions to improve readablity?